src/lib/EventListener/ApiRequestListener.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUi\EventListener;
+
+use eZ\Bundle\EzPublishRestBundle\EventListener\RequestListener;
+use Symfony\Component\HttpFoundation\Request;
+
+class ApiRequestListener extends RequestListener
+{
+    // siteaccesname/api/method_name
+    const ADMIN_UI_REST_PREFIX_PATTERN = '/[a-zA-Z0-9-_]+\/api\/[a-zA-Z0-9-_]+/';
```

Comment:
Hi. This regex pattern is too broad. Upgrading eZ Platform to 7.4 RC broke a wide range of our REST routes in the following formats:

* 'https://example.com/netgen/api/some/path'
* 'https://example.com/admin/netgen/api/some/path'
* 'https://example.com/cro/netgen/api/some/path'

Reply:
Reported as https://jira.ez.no/browse/EZP-29933